themes/admin/Sea_Green/css/sea_green.css

```
@@ -496,93 +494,79 @@ progress {
 
 .lead {
   font-size: calc(1.2675rem + 0.21vw);
+  font-weight: 300;
 }
 @media (min-width: 1200px) {
   .lead {
     font-size: 1.425rem;
   }
 }
```

Stylelint (17.2.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)